Alex_RTOS_Sailboat/RTOS_framework_v1/src/sail_wind.c

```diff
@@ -192,6 +196,7 @@ enum status_code WIND_GetReading(WIND_Reading *reading)
 				break;
 		}
 	} while (found_reading);
+	*/ // I AM COMMENTED OUT!!!
 	
```

Comment:
I think its a good idea to keep the wind part of the WS commented out.

Thoughts?

Reply:
Agreed. At least until we created a new class/file for the wind vane.